### PR TITLE
[feat] logservice: make RPC message max size configurable

### PR DESCRIPTION
**Pull request by volgariver6**
Configure it by new parameter max-message-size in log.toml.

## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [x] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #6238 

## What this PR does / why we need it:

Add a new configuration for logservice to set RPC message size max-message-size, whose default value is 100M.

### CODE REVIEW

**Review by daviszhen — APPROVED** [review]

File: pkg/logservice/client.go [hunk 1]
```diff
@@ -95,6 +95,9 @@ var _ Client = (*managedClient)(nil)
 // to the Log Service in parallel, multiple clients should be created and used
 // to do so.
 func NewClient(ctx context.Context, cfg ClientConfig) (Client, error) {
+	if cfg.MaxMessageSize == 0 {
```
Comment (zhangxu19830126) [inline pkg/logservice/client.go:98]: no default here. morpc has a default value 100MB

File: pkg/logservice/service.go [hunk 2]
```diff
@@ -126,7 +126,8 @@ func NewService(
 	// TODO: check and fix all these magic numbers
 	codec := morpc.NewMessageCodec(mf,
 		morpc.WithCodecPayloadCopyBufferSize(16*1024),
-		morpc.WithCodecEnableChecksum())
+		morpc.WithCodecEnableChecksum(),
+		morpc.WithCodecMaxBodySize(int(cfg.MaxMessageSize)))
```
Comment (zhangxu19830126) [inline pkg/logservice/service.go:130]: use cfg.RPC to config max message, and cn/dn/log has the same config field

**Review by zhangxu19830126 — CHANGES_REQUESTED** [review]

**Review by reusee — APPROVED** [review]

File: pkg/logservice/service.go (hunk 2 above)
Comment (volgariver6) [reply pkg/logservice/service.go]: have removed it.

Comment (volgariver6) [reply pkg/logservice/service.go]: have changed it.

File: pkg/logservice/client.go (hunk 1 above)
Comment (volgariver6) [reply pkg/logservice/client.go]: have removed it.

**Review by zhangxu19830126 — APPROVED** [review]

**Pull request merged**